bigbluebutton-html5/imports/api/cursor/server/modifiers/updateCursor.js

```diff
@@ -25,11 +25,11 @@ export default function updateCursor(meetingId, x = 0, y = 0) {
 
     const { insertedId } = numChanged;
     if (insertedId) {
-      return Logger.info(`Initialized cursor meeting=${meetingId}`);
+      return Logger.debug(`Initialized cursor meeting=${meetingId}`);
     }
 
     if (numChanged) {
-      return Logger.info(`Updated cursor meeting=${meetingId}`);
+      return Logger.verbose(`Updated cursor meeting=${meetingId}`);
```

Comment:
Are you updating other log messages to correspond to their respective level of debugging?